addons/website/views/snippets/snippets.xml

```diff
@@ -345,6 +345,7 @@
 
                 <!-- Newsletter group -->
                 <t id="mass_mailing_newsletter_block_hook"/>
+                <t id="mass_mailing_newsletter_box_hook"/>
```

Comment:
I am putting this one at the end, looks better and consistent with the order of the t-install ones below.